_includes/notebooks/2019-03-01-coupling-resolution.html

```diff
@@ -0,0 +1,123 @@
+<div class="input">
+<div class="prompt input_prompt">In&nbsp;[2]:</div>
+<div class="inner_cell">
+    <div class="input_area">
+<div class=" highlight hl-ipython3"><pre><span></span><span class="k">def</span> <span class="nf">true_theta_height</span><span class="p">(</span><span class="n">level_file</span><span class="p">,</span> <span class="n">orography</span><span class="p">):</span>
+    <span class="sd">&quot;&quot;&quot;</span>
+<span class="sd">    Create a true height field from the UM levels namelist at ``level_file``</span>
+<span class="sd">    and the orography field</span>
+<span class="sd">    &quot;&quot;&quot;</span>
+    <span class="n">target_levels</span> <span class="o">=</span> <span class="n">f90nml</span><span class="o">.</span><span class="n">read</span><span class="p">(</span><span class="n">vertlevs</span><span class="p">)[</span><span class="s1">&#39;VERTLEVS&#39;</span><span class="p">]</span>
+    
+    <span class="n">eta</span> <span class="o">=</span> <span class="n">numpy</span><span class="o">.</span><span class="n">array</span><span class="p">(</span><span class="n">target_levels</span><span class="p">[</span><span class="s1">&#39;eta_theta&#39;</span><span class="p">])</span>
+    <span class="n">const_lev</span> <span class="o">=</span> <span class="n">target_levels</span><span class="p">[</span><span class="s1">&#39;first_constant_r_rho_level&#39;</span><span class="p">]</span><span class="o">-</span><span class="mi">1</span>
+
+    <span class="n">Zsea</span> <span class="o">=</span> <span class="n">target_levels</span><span class="p">[</span><span class="s1">&#39;z_top_of_model&#39;</span><span class="p">]</span> <span class="o">*</span> <span class="n">eta</span>
+    <span class="n">C</span> <span class="o">=</span> <span class="p">(</span><span class="mi">1</span> <span class="o">-</span> <span class="n">eta</span><span class="o">/</span><span class="n">eta</span><span class="p">[</span><span class="n">const_lev</span><span class="p">])</span><span class="o">**</span><span class="mi">2</span>
+    <span class="n">C</span><span class="p">[</span><span class="n">const_lev</span><span class="p">:]</span> <span class="o">=</span> <span class="mi">0</span>
+    <span class="n">Z</span> <span class="o">=</span> <span class="n">target_Zsea</span><span class="p">[:,</span> <span class="n">numpy</span><span class="o">.</span><span class="n">newaxis</span><span class="p">,</span> <span class="n">numpy</span><span class="o">.</span><span class="n">newaxis</span><span class="p">]</span> <span class="o">+</span> <span class="n">numpy</span><span class="o">.</span><span class="n">multiply</span><span class="o">.</span><span class="n">outer</span><span class="p">(</span><span class="n">target_C</span><span class="p">,</span><span class="n">orog</span><span class="p">)</span>
+
+    <span class="k">return</span> <span class="n">Z</span>
+</pre></div>
+
+</div>
+</div>
+</div>
+
+</div>
+ 
```

Comment:
Do you want to have a closing line not to hit a sudden halt? E.g. "And then it's done!"